### ISSUE TITLE
Better contrast for accessibility

### ISSUE THREAD

**Issue by jackherizsmith** [fac19/week4-harl#47]
Names could be better colour contrast (fails WCAG AAA test for 7:1)

**Comment by akomiqaia** [comment]
what names?
